test/deploy/kubernetes/3rd_party_services/deploy-3rd-party-comp.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+namespace="default"
```

Comment:
Change namespace to `cortx`

Reply:
right now I kept it default and all hardcode default is change to $namespace variable , once we finalize the namespace we can update the var